src/coord/src/catalog.rs

```diff
@@ -444,6 +482,15 @@ pub struct Role {
     pub oid: u32,
 }
 
+#[derive(Debug, Serialize, Clone)]
+pub struct ComputeInstance {
+    pub name: String,
+    pub id: usize,
```

Comment:
`usize` seems to be used throughout rather than `ComputeInstanceId`, which I recommend swapping in. It should be easy to track them down, by toggling the typedef to be `isize` instance and track down the errors.

Reply:
Yep--just some typing I hadn't gotten to just yet. Thanks for the nudge.